### ISSUE TITLE
Audio web socket

### ISSUE THREAD

**Issue by avnd98** [RetellAI/python-backend-demo#7]
Does retell supports any other call service provider apart from twilio?

**Comment by weijia-yu** [comment]
Yes, which one are you using?

**Comment by kesbeast23** [comment]
Good day,how about asterisk?
https://www.asterisk.org/

**Comment by tkejr** [comment]
I think you can configure any provider

**Comment by fedorkar** [comment]
Yes. Retell supports imported numbers and custom telephony integrations beyond Twilio. The current setup guidance is at https://docs.retellai.com/deploy/custom-telephony. Since the original question was answered and the old audio-WebSocket path is no longer the recommended integration, we are closing this issue.
